docs/examples/extension.ipynb

```diff
@@ -11,14 +11,14 @@
    "cell_type": "markdown",
```

Comment:
<p>Add a hyperlink to the docs! Also, note that one doesn't <em>need</em> to implement <code>_h</code> or <code>_p</code>   -- they just make the calculation faster.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/steven-murray/halomod/pull/60/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>